readme.md

```diff
@@ -4,7 +4,12 @@
 
 This userChrome.css mimics the [elementary GTK+ theme](https://github.com/elementary/stylesheet) to make firefox blend in better in elementary OS.
 
-## How to use
+## How to install
+### Terminal
+```shell
+curl https://git.io/fYMYZ -o "$HOME/.mozilla/firefox/$( cat $HOME/.mozilla/firefox/profiles.ini | grep -oP '(?<=Path=).*' )/chrome/userChrome.css" -sLS --create-dirs
```

Comment:
I think this will not work well if the user has more than one profile.

Reply:
Correct. This will just affect the active user profile.

Reply:
Not really. The grep will print each profile on a separate line, which will create a really bizarre path.

Reply:
Good to know! Is there a pattern to the profiles? Will the current one be the first line? Is there another mechanism to indicate the current profile?

Reply:
My profiles.ini files looks like this:

    [General]
    StartWithLastProfile=1
    
    [Profile0]
    Name=default
    IsRelative=1
    Path=jfkdaj.edited
    Default=1
    
    [Profile1]
    Name=dev-edition-default
    IsRelative=1
    Path=jdflakj.edited

So yes, the default profile is indicated by "Default=1". But I wouldn't count on the exact order of the values in a profile, so this might be hard to do with grep.

Reply:
Thanks for that. Looks like we need some more logic than a one liner. I'll look into a more robust method.